app/controllers/users/registrations_controller.rb

```diff
@@ -2,16 +2,20 @@
 
 module Users
   class RegistrationsController < Devise::RegistrationsController
+    include Trackable
+
     before_action :configure_sign_up_params, only: [:create]
     before_action :configure_account_update_params, only: [:update]
     after_action :remove_notice, only: %i[create]
+    after_action -> { track_event("Viewed Sign Up") }, only: %i[new]
 
     def create
       super do |resource|
         if resource.persisted?
           resource.create_account
           resource.setup_email_workers
           resource.send_welcome_email
+          track_event("New Sign Up")
```

Comment:
Couldn't we write this one as after_action for the create action?

Reply:
Yes wasn't sure if I can use `resource.persisted` in the hook, but looks like it works. So here is the new commit: b6571fd